tool/state-processor/src/system/mod.rs

```diff
@@ -125,54 +125,25 @@ impl Processor {
 				// If the evm account is a contract contract with sufficients, then we should
 				// increase the sufficients by one.
 				if self.solo_state.contains_key(&full_key(
-					b"EVM",
+					b"Evm",
```

Comment:
This is incorrect. `EVM` in darwinia 1.0's chain and `Evm` in darwinia 2.0's chain.